felix/fv/test-connection/test-connection.go

```diff
@@ -148,6 +149,16 @@ func main() {
 		log.WithError(err).Fatal("Invalid --stdin")
 	}
 
+	timeout := 0
+
+	if toval := arguments["--timeout"]; toval != nil {
+		timeout, err = strconv.Atoi(toval.(string))
+		if err != nil {
+			// panic on error
+			log.WithField("timeout", timeout).Fatal("Invalid duration argument")
```

Comment:
```suggestion
			log.WithField("timeout", timeout).Fatal("Invalid --timeout argument")
```
It's explicitly not a time.Duration here so probably best not to use that word